lib/network/modules/components/NavigationHandler.js

```diff
@@ -160,7 +160,12 @@ class NavigationHandler {
 
     this.body.view.scale = scale;
     this.body.view.translation = { x: tx, y: ty };
-    this.body.emitter.emit('zoom', { direction: '+', scale: this.body.view.scale, pointer: pointer });
+    if (typeof(pointer) != 'undefined') {
```

Comment:
Why do we need this conditional expression here? There is no `pointer` in the scope, so this expression always evaluates to `false`, right?